includes/wc-formatting-functions.php

```diff
@@ -1351,6 +1351,23 @@ function wc_implode_html_attributes( $raw_attributes ) {
 	return implode( ' ', $attributes );
 }
 
+/**
+ * Escape JSON for use on HTML or attribute text nodes.
+ *
+ * @since 3.5.5
```

Comment:
WC versions in these tags should be prefixed:

```suggestion
 * @since WC-3.5.5
```